### DISCUSSION TITLE
PETITION FOR MORE TRANSPARENCY

### DISCUSSION THREAD

**Discussion by timokoenig** [Digitaler-Impfnachweis/covpass-ios#132] [General]
# PETITION FOR MORE TRANSPARENCY

From a community perspective, the development of CovPass is like a big black box and the product that comes out is not great. Features do not work and do not get fixed even though the community presents simple solutions that can be easily adopted. Community work like discussions, tickets, and pull requests are not welcomed or being ignored. The average response time is between 2 weeks and never.

We all like the idea behind CovPass and want it to succeed and for it to be used everywhere, but that is only possible with some changes.

## What We Want

- Bug requests should be taken seriously
- Questions should be answered in a reasonable time
- Documentation of why certain decisions have been made
- Document update process of domestic and EU rules
- Clarification if community work is welcomed
  - if yes, list contact persons that are available for questions and active on Github
  - including the community in the current development could improve the quality of the final product

## Signed

Timo Koenig 👑

**Comment by Ein-Tim** [comment]
I support this.

**Comment by timokoenig** [comment]
too bad... I guess no answer is also an answer.

**Comment by alexcimander** [comment]
Hello everyone,

As you all know and have experienced, the last few weeks have been anything but easy. Constantly changing regulations, new admission concepts, inconsistencies between the federal states and inconsistencies in the EU, a change of government, a pandemic that is also affecting our teams, to name just a few challenges.

Due to these many topics, the exchange with you was unfortunately too short: You are right, it doesn't work that way! 
Your active participation and your unparalleled commitment deserve more attention and our communication should be more transparent.

Therefore, we have planned comprehensive process changes within, which should bring you and us closer together and promote a common further development.

In the coming days we would like to actively participate in discussions again, work together on your bug-findings and enhancements and last but not least communicate more transparently in your direction in form of announcements, rule updates etc.

We thank you for your support in the last months and look forward to future cooperation.

Your CovPass(Check) Team 💯

**Reply by timokoenig** [reply]
Even though I haven't seen any improvements, is CovPass addressing the most critical issue that the DE entry and domestic rules are still wrong for over three month now? I can't imagine that RKI is fine that the app does not even comply with their own rules.

**Reply by aherberger** [reply]
The "hidden" mode of the RKI/development team is from various points of view problematic. Nobody knews the details of "ownership" of this project - if its an open source project, that it should be managed as one. Currently its a closed source project with publishing of the source code.... - so what makes people here in github "responsible" to "own" the project? 
So I support any initiative to provide more transparency.
